docs/langfuse.md

```diff
@@ -0,0 +1,119 @@
+## Local deployment
+
+### Set up Redis
+
+A local instance of LangFuse will bring it's own instance of `Redis` with some basic authentication setup. The password defaults to `myredissecret` (if you do not specify the env var `REDIS_AUTH`) and LangFuse will automatically use the database number `0` of that Redis instance (remember that it comes by default with 16 databases, 0 to 15).
+If you also run a local instance of `Redis` for the `Companion`, you have two options.
+a) You let the `Companion` use the Redis instance that comes with Langfuse and addressing database `1` by adding these lines in your `config/config.json`:
```

Comment:
```suggestion

1. You let the Companion use the Redis instance that comes with LangFuse and addressing database `1` by adding these lines in your `config/config.json`:
```
To double-check if I get it right: the Redis instance comes with LangFuse and database `1`? What is it addressing? Did you mean "corresponding"?

Reply:
LangFuse comes with Redis. Redis has multiple Databases, 0 to 16. We want to set the Companion to use database `1`.
1. You let the Companion use the Redis instance that comes with LangFuse and addressing database `1` by adding these lines in your `config/config.json`